pyastroweatherio/dataclasses.py

```diff
@@ -71,8 +75,12 @@ def seeing(self) -> int:
 
     @property
     def seeing_percentage(self) -> int:
-        """Return Seeing."""
-        return int((100 + 100 / 7 - self._seeing * 100 / 7))
+        """Return Seeing.
+        Plain value is an int between 1 and 8 (1 is below .5", 8 is above 2.5"; 1 is best;
+        see https://github.com/Yeqzids/7timer-issues/wiki/Wiki#astro),
+        converting it to value between 0 and 100.
+        """
+        return int(100 - 100 * (self._seeing - 1 ) / 7)
```

Comment:
Mathematically it's the same, but I find my version easier to read. I can revert it if you do not agree.